airflow/models/baseoperator.py

```diff
@@ -1734,6 +1721,8 @@ def __repr__(self) -> str:
     params: Union[ParamsDict, dict] = attr.ib(factory=ParamsDict)
     template_fields: Iterable[str] = attr.ib()
 
+    subdag: None = attr.ib(init=False)
```

Comment:
Is this used?

Reply:
"badly" as part of the serialization code https://github.com/apache/airflow/blob/14a057ff921d7e6ceb70326f5fecac29d3a093ad/airflow/serialization/serialized_objects.py#L970-L971

Reply:
May be fixed/not needed by your #20945?